index.js

```diff
@@ -175,6 +184,19 @@ alexa.response = function(session) {
   };
 };
 
+alexa.directives = function(directives) {
+  // load the alexa response directives information into details
+  this.details = directives;
+
+  this.set = function(directive) {
+    this.details.push(directive);
+  };
+
+  this.clear = function() {
```

Comment:
Sorry been a bit busy. I can answer specific questions, but won't be able to focus on it. Keep at it until it's green!